src/config/dark-sites.config

```diff
@@ -836,6 +837,7 @@ thealiendrew.github.io
 thegraph.com
 thelinuxcast.org
 themes.vscode.one
+thestanleyparable.fandom.com
```

Comment:
Not dark-by-default:

![image](https://user-images.githubusercontent.com/25481501/187212363-bb5188ce-4f0c-49e1-acb6-9f9525af39fc.png)